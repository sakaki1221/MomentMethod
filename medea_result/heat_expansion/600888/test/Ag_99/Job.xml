<?xml version='1.0'?>
<!DOCTYPE MDRESULTS PUBLIC "-//MD//DTD MDRESULTS 1.0" "http://materialsdesign.com/DTD/mdresults_1_0.dtd" []>

<MDRESULTS>
  <System id="initial">
    <Date>16 Jan 2017 05:57:59 GMT</Date>
    <Characterization>
      <PropertyValue property="symmetry" type="text" units="">P1</PropertyValue>
      <PropertyValue property="setting" type="text" units="">P1</PropertyValue>
      <PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue>
      <PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue>
      <PropertyValue property="primitive Z" type="text" units="">32</PropertyValue>
    <PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization>
  </System>
  <Calculation id="initial">
    <Date>16 Jan 2017 05:57:59 GMT</Date>
    <System>initial</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-85.370024</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">1.575100</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">707.460</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-10.030</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">10.03000 10.03000 10.03000 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">15.4213</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-118.1162</PropertyValue>
    <PropertyValue property="VASP energy" type="float" units="eV">-85.370024</PropertyValue><PropertyValue property="Fermi energy" type="float" units="eV">1.575100</PropertyValue><PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue><PropertyValue property="cell volume" type="float" units="Ang^3">707.460</PropertyValue><PropertyValue property="pressure" type="float" units="GPa">-10.030</PropertyValue><PropertyValue dimensions="6" property="stress" type="float" units="GPa">10.03000 10.03000 10.03000 0.00000 0.00000 0.00000</PropertyValue><PropertyValue property="energy of formation" type="float" units="kJ/mol">15.4213</PropertyValue><PropertyValue property="heat of formation" type="float" units="kJ/mol">-118.1162</PropertyValue></Results>
  </Calculation>
<System id="displaced_Ag__a"><Date>18 Jan 2017 03:06:40 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__a">
    <Date>18 Jan 2017 03:06:40 GMT</Date>
    <System>displaced_Ag__a</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-85.369808</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">1.575000</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">50459935.810 6172138.085 0.000
214941996.743 272223774.489 0.000
214942006.726 4.991 -272223774.489
50459936.037 -6172138.085 0.000
-76014753.258 866265.010 -866264.994
-67893518.936 0.000 0.000
50893068.526 -5739005.588 0.000
214941996.743 -4.991 272223774.489
214942006.726 -272223774.489 0.000
50459936.037 0.113 -6172138.085
-76014753.290 866264.979 866264.994
-2283474525.232 0.000 0.000
-76014753.227 -866264.979 -866264.994
974548.119 0.000 0.000
247535227.867 -311855397.983 0.000
247535216.431 -5.718 311855397.983
50893068.316 5739005.588 0.000
118461737.984 -974548.137 974548.119
-116187792.374 0.000 0.000
50459935.810 -0.113 6172138.085
-76014753.258 -866265.010 866264.994
-67893518.936 0.000 0.000
247535227.867 5.718 -311855397.983
247535216.431 311855397.983 0.000
50893068.316 -0.105 5739005.588
118461738.020 -974548.101 -974548.119
253165944.613 0.000 0.000
118461737.948 974548.101 974548.119
-27937046.069 0.000 0.000
50893068.526 0.105 -5739005.588
118461737.984 974548.137 -974548.119
-116187792.374 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">707.460</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-9.976</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">9.97500 9.97700 9.97700 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">15.4220</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-117.3966</PropertyValue>
    </Results>
  </Calculation><System id="displaced_Ag__b"><Date>18 Jan 2017 03:06:41 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__b">
    <Date>18 Jan 2017 03:06:41 GMT</Date>
    <System>displaced_Ag__b</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-85.369808</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">1.575000</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">-50893068.316 -5739005.588 0.000
-247968348.916 -312505096.729 0.000
-247968360.376 -5.730 312505096.729
-50893068.526 5739005.588 0.000
76123036.383 757981.884 -757981.870
67568669.563 0.000 0.000
-50351652.915 6280421.209 0.000
-247968348.916 5.730 -312505096.729
-247968360.376 312505096.729 0.000
-50893068.526 -0.105 5739005.588
76123036.355 757981.856 757981.870
2289646663.317 0.000 0.000
76123036.410 -757981.856 -757981.870
-1624246.864 0.000 0.000
-215591705.482 272765190.111 0.000
-215591695.479 5.001 -272765190.111
-50351652.684 -6280421.209 0.000
-118678304.232 -866265.010 866264.994
116079509.249 0.000 0.000
-50893068.316 0.105 -5739005.588
76123036.383 -757981.884 757981.870
67568669.563 0.000 0.000
-215591705.482 -5.001 272765190.111
-215591695.479 -272765190.111 0.000
-50351652.684 0.115 -6280421.209
-118678304.201 -866264.979 -866264.994
-254140492.731 0.000 0.000
-118678304.264 866264.979 866264.994
28478461.691 0.000 0.000
-50351652.915 -0.115 6280421.209
-118678304.232 866265.010 -866264.994
116079509.249 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">707.460</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-9.975</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">9.97500 9.97600 9.97600 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">15.4220</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-117.3833</PropertyValue>
      <PropertyValue property="zero-point energy" type="float" units="kJ/mol">0.8990</PropertyValue>
      <PropertyValue property="E(0)" type="float" units="kJ/mol">16.3203</PropertyValue>
      <PropertyValue property="E(PV)" type="float" units="kJ/mol">-133.5376</PropertyValue>
      <PropertyValue property="Eelec" type="float" units="kJ/mol">15.4213</PropertyValue>
      <PropertyValue property="Temperatures" type="float" units="K">0.001 1 2 3 4 5 10 15 20 30 40 50 60 70 80 90 100 125 150 175 200 225 250 273 275 298 300 350 400 450 500 550 600 650 700 750 800 850 900 950 1e+003 1.05e+003 1.1e+003 1.15e+003 1.2e+003 1.25e+003 1.3e+003 1.35e+003 1.4e+003 1.45e+003 1.5e+003 1.55e+003 1.6e+003 1.65e+003 1.7e+003 1.75e+003 1.8e+003 1.85e+003 1.9e+003 1.95e+003 2e+003 2.1e+003 2.2e+003 2.3e+003 2.4e+003 2.5e+003 2.6e+003 2.7e+003 2.8e+003 2.9e+003 3e+003</PropertyValue>
      <PropertyValue property="S(T)" type="float" units="J/K/mol">0.0000 0.0001 0.0027 0.0100 0.0226 0.0425 0.4789 1.9058 4.1254 9.3438 14.3734 18.8416 22.7609 26.2153 29.2875 32.0458 34.5443 39.9093 44.3469 48.1252 51.4125 54.3205 56.9272 59.1078 59.2887 61.2811 61.4471 65.2753 68.5951 71.5255 74.1482 76.5216 78.6889 80.6831 82.5298 84.2492 85.8578 87.3690 88.7938 90.1418 91.4206 92.6370 93.7970 94.9053 95.9666 96.9845 97.9625 98.9037 99.8106 100.6857 101.5312 102.3489 103.1407 103.9081 104.6526 105.3755 106.0781 106.7615 107.4266 108.0744 108.7058 109.9228 111.0830 112.1917 113.2532 114.2713 115.2496 116.1909 117.0980 117.9733 118.8188</PropertyValue>
      <PropertyValue property="Cv(T)" type="float" units="kJ/mol">0.0000 0.0006 0.0097 0.0295 0.0628 0.1236 1.7918 5.7080 9.8350 15.7567 19.0437 20.9073 22.0326 22.7543 23.2413 23.5841 23.8339 24.2249 24.4412 24.5730 24.6592 24.7185 24.7610 24.7903 24.7926 24.8149 24.8166 24.8502 24.8720 24.8870 24.8978 24.9057 24.9118 24.9165 24.9202 24.9232 24.9257 24.9277 24.9294 24.9309 24.9321 24.9332 24.9341 24.9349 24.9356 24.9363 24.9368 24.9373 24.9377 24.9381 24.9385 24.9388 24.9391 24.9394 24.9396 24.9399 24.9401 24.9403 24.9404 24.9406 24.9407 24.9410 24.9412 24.9414 24.9416 24.9418 24.9419 24.9420 24.9421 24.9422 24.9423</PropertyValue>
      <PropertyValue property="H(T)-Eelec" type="float" units="kJ/mol">-132.6386 -132.6386 -132.6386 -132.6385 -132.6385 -132.6384 -132.6348 -132.6165 -132.5775 -132.4468 -132.2712 -132.0706 -131.8554 -131.6312 -131.4011 -131.1669 -130.9298 -130.3285 -129.7199 -129.1071 -128.4917 -127.8744 -127.2559 -126.6860 -126.6365 -126.0660 -126.0163 -124.7746 -123.5315 -122.2875 -121.0429 -119.7978 -118.5524 -117.3067 -116.0608 -114.8147 -113.5685 -112.3221 -111.0757 -109.8292 -108.5826 -107.3360 -106.0893 -104.8426 -103.5959 -102.3491 -101.1023 -99.8554 -98.6085 -97.3616 -96.1147 -94.8678 -93.6209 -92.3739 -91.1269 -89.8800 -88.6330 -87.3860 -86.1390 -84.8919 -83.6449 -81.1508 -78.6568 -76.1626 -73.6685 -71.1743 -68.6802 -66.1860 -63.6918 -61.1976 -58.7034</PropertyValue>
      <PropertyValue property="E(T)-Eelec" type="float" units="">0.8990 0.8990 0.8990 0.8990 0.8991 0.8992 0.9028 0.9210 0.9601 1.0908 1.2664 1.4670 1.6821 1.9063 2.1364 2.3707 2.6078 3.2091 3.8176 4.4304 5.0459 5.6632 6.2817 6.8515 6.9011 7.4716 7.5212 8.7630 10.0060 11.2500 12.4947 13.7397 14.9852 16.2309 17.4768 18.7229 19.9691 21.2154 22.4619 23.7084 24.9549 26.2016 27.4482 28.6950 29.9417 31.1885 32.4353 33.6822 34.9290 36.1759 37.4228 38.6698 39.9167 41.1637 42.4106 43.6576 44.9046 46.1516 47.3986 48.6456 49.8927 52.3867 54.8808 57.3749 59.8691 62.3632 64.8574 67.3516 69.8458 72.3400 74.8342</PropertyValue>
    </Results>
  </Calculation></MDRESULTS>
