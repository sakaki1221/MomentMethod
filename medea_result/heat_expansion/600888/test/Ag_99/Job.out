Startup is 'sourced'
Opening the database
Phonon calculation for Untitled

Phonon parameters
=================
The calculation explores the full Brillouin zone and accounts for an interaction range of about 10.0 Angstroem.
The asymmetric atoms are displaced by +/-0.02 Angstroem.

In a first step, need to run an optimization of atom positions for the supercell containing 32 atoms.
Then, need to run a total number of 2 single point energy calculations on supercells with displaced atoms containing 32 atoms.

      Supercell transformation matrix              Supercell lattice vectors
       2.00000    0.00000    0.00000             8.91047    0.00000    0.00000
       0.00000    2.00000    0.00000             0.00000    8.91047    0.00000
       0.00000    0.00000    2.00000             0.00000    0.00000    8.91047

     Supercell           Site        Displaced atom     Displacement (fractional units)
   ---------------    ----------     --------------     -------------------------------
  undisplaced           ----            ----             0.00000    0.00000    0.00000
  displaced_Ag__a       Ag              Ag               0.00225    0.00000    0.00000
  displaced_Ag__b       Ag              Ag              -0.00225    0.00000    0.00000

Invariance conditions are not enforced.
No approximation to LO/TO splitting
Properties: Phonon dispersion and form factors 
            Phonon density of states 
            Thermodynamic functions
==========================================


VASP parameters
===============
Since no magnetic moments are in the model, this is a non-magnetic calculation using 'accurate' precision
and a user-defined planewave cutoff energy of 600.000 eV.

The electronic iterations convergence is 1.00E-005 eV using the Very Fast (RMM-DIIS) algorithm
and reciprocal space projection operators.

The requested k-spacing is 0.1 per Angstrom, which leads to a 25x25x25 mesh.
This corresponds to actual k-spacings of 0.098 x 0.098 x 0.098 per Angstrom.

Using first order Methfessel-Paxton smearing with a width of 0.2 eV.

Other non-default parameters:
	Extrafine augmentation grid for accurate forces is TRUE
	Algorithm is Very Fast (RMM-DIIS)
	Wave functions is TRUE
==========================================

------------------------------------------------------------------------

Results for initial structural minimization of the supercell without displacements:
--------------------------------------------------------------------------------------

There are 20 symmetry-unique k-points
The plane wave cutoff is 600.00 eV

   VASP energy:             -85.370024 eV for Ag32

   Initial VASP energy:     -85.370024 eV for Ag32
   Relaxation energy:         0.000000 eV gained after 1 optimization steps.

Electronic contributions:
                          Empirical Formula          Cell
                                    Ag          (Ag)32
                          --------------- ---------------
              VASP Energy        -257.405       -8236.953 kJ/mol
      Energy of formation          15.421         493.482
                  PV term        -133.538       -4273.202

   This is the electronic part of the formation energy with respect to the
   elements in their standard state. It is obtained from the total energy (above)
   and tabulated VASP energies for the elements in their standard state.
   See the documentation for more details.

   Details for Ag:
                                             VASP energy (Etot):     -257.40 kJ/mol
       Standard state energy of 1.00 atoms of Ag  1.00 * -272.83:  -  -272.83
                                                                   ----------
                                        Energy of formation (Ef):       15.42 kJ/mol


Cell parameters:
 Parameter    Value    
---------- ------------
         a     8.910466
         b     8.910466
         c     8.910466
     alpha    89.999999
      beta    89.999999
     gamma    89.999999

    Volume    88.432368 Ang^3

        Density:     8.102 Mg/m^3

   The pressure given below is exerted by the system according to its volume.
   Positive pressure would cause expansion during full geometry optimization.
       Pressure:   -10.030 GPa
               =  -100.300 kbar

                      XX        YY        ZZ        XY        YZ        ZX
         Stress:    10.030    10.030    10.030     0.000     0.000     0.000 GPa
               =   100.300   100.300   100.300     0.000     0.000     0.000 kbar
   The stress tensor above is imposed on the system, i.e. negative values of diagonal components 
   would cause expansion of the corresponding lattice parameter upon full geometry optimization.
   The pressure and stress include only electronic terms, i.e. the vibrational,
   temperature and other terms are not included here.

Fractional Coordinates:
	Atom          Initial Coordinates             Final Coordinates
	-----      -------- -------- --------      -------- -------- --------
	Ag           0.7500   0.7500   0.5000        0.7500   0.7500   0.5000
	Ag           0.7500   0.7500   0.0000        0.7500   0.7500   0.0000
	Ag           0.7500   0.0000   0.2500        0.7500   0.0000   0.2500
	Ag           0.7500   0.2500   0.5000        0.7500   0.2500   0.5000
	Ag           0.0000   0.7500   0.2500        0.0000   0.7500   0.2500
	Ag           0.0000   0.0000   0.5000        0.0000   0.0000   0.5000
	Ag           0.2500   0.7500   0.5000        0.2500   0.7500   0.5000
	Ag           0.7500   0.0000   0.7500        0.7500   0.0000   0.7500
	Ag           0.7500   0.2500   0.0000        0.7500   0.2500   0.0000
	Ag           0.7500   0.5000   0.2500        0.7500   0.5000   0.2500
	Ag           0.0000   0.7500   0.7500        0.0000   0.7500   0.7500
	Ag           0.0000   0.0000   0.0000        0.0000   0.0000   0.0000
	Ag           0.0000   0.2500   0.2500        0.0000   0.2500   0.2500
	Ag           0.0000   0.5000   0.5000        0.0000   0.5000   0.5000
	Ag           0.2500   0.7500   0.0000        0.2500   0.7500   0.0000
	Ag           0.2500   0.0000   0.2500        0.2500   0.0000   0.2500
	Ag           0.2500   0.2500   0.5000        0.2500   0.2500   0.5000
	Ag           0.5000   0.7500   0.2500        0.5000   0.7500   0.2500
	Ag           0.5000   0.0000   0.5000        0.5000   0.0000   0.5000
	Ag           0.7500   0.5000   0.7500        0.7500   0.5000   0.7500
	Ag           0.0000   0.2500   0.7500        0.0000   0.2500   0.7500
	Ag           0.0000   0.5000   0.0000        0.0000   0.5000   0.0000
	Ag           0.2500   0.0000   0.7500        0.2500   0.0000   0.7500
	Ag           0.2500   0.2500   0.0000        0.2500   0.2500   0.0000
	Ag           0.2500   0.5000   0.2500        0.2500   0.5000   0.2500
	Ag           0.5000   0.7500   0.7500        0.5000   0.7500   0.7500
	Ag           0.5000   0.0000   0.0000        0.5000   0.0000   0.0000
	Ag           0.5000   0.2500   0.2500        0.5000   0.2500   0.2500
	Ag           0.5000   0.5000   0.5000        0.5000   0.5000   0.5000
	Ag           0.2500   0.5000   0.7500        0.2500   0.5000   0.7500
	Ag           0.5000   0.2500   0.7500        0.5000   0.2500   0.7500
	Ag           0.5000   0.5000   0.0000        0.5000   0.5000   0.0000

Results for supercell calculations:
------------------------------------------------------------------------
	        Supercell         Energy (eV)
	      -------------      -------------
	        undisplaced       -85.37002
	     displaced_Ag__a      -85.36981
	     displaced_Ag__b      -85.36981


Symmetry analysis of Gamma point phonon frequencies:
----------------------------------------------------
   Crystal  Point  Group: O_h   (m3m) 
   Supercell Point Group: O_h   (m3m) 
    Table of irreducible representations of point group:
 ------------------------------------------
    IR     1  5  7  9 11  6  8 10 12 13 16 17 18 21 23 14 15 19 20 22 24  2  3  4 25 38 39 43 44 46 48 29 30 31 32 33 34 35 36 26 27 28 37 40 41 42 45 47
 ------------------------------------------
 A1g(R)    1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1   x^2+y^2+z^2                         
 A2g       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1                                       
 Eg(R)     2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2  2  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1  2  2  2  0  0  0  0  0  0   (2z^2-x^2-y^2, x^2-y^2)             
 T1g       3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1  3  1  1  1  1  1  1  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 T2g(R)    3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1  3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0 -1 -1 -1  1  1  1  1  1  1   (xy,xz,yz)                          
 A1u       1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 A2u       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1 -1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1                                       
 Eu        2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2 -2  0  0  0  0  0  0  1  1  1  1  1  1  1  1 -2 -2 -2  0  0  0  0  0  0                                       
 T1u(I)    3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1 -3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  1  1  1  1  1  1  1  1  1   (x,y,z)                             
 T2u       3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -3  1  1  1  1  1  1  0  0  0  0  0  0  0  0  1  1  1 -1 -1 -1 -1 -1 -1                                       
 ------------------------------------------
    Wave vector= 0.0000  0.0000  0.0000    no macro. electric field
 Multi. Omega   Irreducible Representations
  3     0.017     T1u(I)                                                                                                     
Notation: (R)  Raman active mode
          (I)  Infrared active mode
          (RI) Raman and Infrared active mode

Thermodynamic Properties
   for empirical formula Ag

The following thermodynamic functions are obtained from the phonon
density of states within the harmonic approximation. The results are
meaningful only below the melting point, which lies well below 3000 K
for most materials.
Definitions:
      Cv            : vibrational heat capacity at constant volume
E_vib(T)=E(T)-E(0)  : the change in vibrational internal energy from 0 K, where
                        E(0) = E_elec + ZPE is the electronic energy of formation E_elec plus the zero point energy ZPE
      S(T)          : the vibrational entropy at temperature T
   -(A(T)-E(0))     : the change in the vibrational Helmholtz free energy from 0 K
      E(T)          : the electronic energy of formation plus the vibrational internal energy, so this term is
                        E_elec + ZPE + E_vib(T)
      A(T)          : the electronic plus vibrational Helmholtz free energy,
                        E(T) - T.S(T)
Note that the electronic energy, which is part of the enthalpy and free energies,
 is referenced to the elements in their standard state.

Electronic energy =    15.42 kJ/mol (referenced to elements in the standard state)
          PV term =  -133.54 kJ/mol
Zero-point energy =     0.90 kJ/mol

    T       Cv       E(T)-E(0)      S(T)   -(A(T)-E(0))     E(T)        A(T)
    K     J/K/mol      kJ/mol     J/K/mol     kJ/mol       kJ/mol      kJ/mol
  ----- ----------- ----------- ----------- ----------- ----------- -----------
     0      0.0000      0.0000      0.0000      0.0000     16.3203     16.3203
     1      0.0006      0.0000      0.0001      0.0000     16.3203     16.3203
     2      0.0097      0.0000      0.0027      0.0000     16.3203     16.3203
     3      0.0295      0.0000      0.0100      0.0000     16.3203     16.3203
     4      0.0628      0.0001      0.0226      0.0000     16.3204     16.3203
     5      0.1236      0.0002      0.0425      0.0001     16.3205     16.3203
    10      1.7918      0.0038      0.4789      0.0010     16.3241     16.3193
    15      5.7080      0.0220      1.9058      0.0066     16.3423     16.3137
    20      9.8350      0.0611      4.1254      0.0214     16.3814     16.2989
    30     15.7567      0.1918      9.3438      0.0885     16.5121     16.2318
    40     19.0437      0.3674     14.3734      0.2075     16.6877     16.1128
    50     20.9073      0.5680     18.8416      0.3741     16.8883     15.9462
    60     22.0326      0.7831     22.7609      0.5825     17.1035     15.7378
    70     22.7543      1.0073     26.2153      0.8277     17.3276     15.4926
    80     23.2413      1.2374     29.2875      1.1055     17.5578     15.2148
    90     23.5841      1.4717     32.0458      1.4124     17.7920     14.9079
   100     23.8339      1.7088     34.5443      1.7456     18.0291     14.5747
   125     24.2249      2.3101     39.9093      2.6786     18.6304     13.6418
   150     24.4412      2.9186     44.3469      3.7334     19.2390     12.5870
   175     24.5730      3.5314     48.1252      4.8904     19.8518     11.4299
   200     24.6592      4.1469     51.4125      6.1355     20.4672     10.1848
   225     24.7185      4.7642     54.3205      7.4579     21.0845      8.8624
   250     24.7610      5.3827     56.9272      8.8490     21.7030      7.4713
   273     24.7903      5.9525     59.1078     10.1838     22.2728      6.1365
   275     24.7926      6.0021     59.2887     10.3022     22.3224      6.0181
   298     24.8149      6.5726     61.2811     11.6890     22.8929      4.6313
   300     24.8166      6.6222     61.4471     11.8118     22.9426      4.5086
   350     24.8502      7.8640     65.2753     14.9822     24.1843      1.3381
   400     24.8720      9.1071     68.5951     18.3308     25.4274     -2.0105
   450     24.8870     10.3510     71.5255     21.8352     26.6714     -5.5149
   500     24.8978     11.5957     74.1482     25.4782     27.9160     -9.1579
   550     24.9057     12.8408     76.5216     29.2459     29.1611    -12.9255
   600     24.9118     14.0862     78.6889     33.1269     30.4065    -16.8066
   650     24.9165     15.3319     80.6831     37.1118     31.6522    -20.7915
   700     24.9202     16.5778     82.5298     41.1927     32.8981    -24.8724
   750     24.9232     17.8239     84.2492     45.3626     34.1442    -29.0423
   800     24.9257     19.0701     85.8578     49.6157     35.3904    -33.2954
   850     24.9277     20.3164     87.3690     53.9467     36.6368    -37.6264
   900     24.9294     21.5629     88.7938     58.3511     37.8832    -42.0308
   950     24.9309     22.8094     90.1418     62.8248     39.1297    -46.5045
  1000     24.9321     24.0559     91.4206     67.3641     40.3762    -51.0438
  1050     24.9332     25.3026     92.6370     71.9657     41.6229    -55.6454
  1100     24.9341     26.5492     93.7970     76.6268     42.8695    -60.3065
  1150     24.9349     27.7960     94.9053     81.3445     44.1163    -65.0242
  1200     24.9356     29.0427     95.9666     86.1165     45.3630    -69.7962
  1250     24.9363     30.2895     96.9845     90.9404     46.6098    -74.6201
  1300     24.9368     31.5363     97.9625     95.8142     47.8566    -79.4939
  1350     24.9373     32.7832     98.9037    100.7360     49.1035    -84.4157
  1400     24.9377     34.0300     99.8106    105.7039     50.3504    -89.3836
  1450     24.9381     35.2769    100.6857    110.7165     51.5972    -94.3961
  1500     24.9385     36.5238    101.5312    115.7719     52.8442    -99.4516
  1550     24.9388     37.7708    102.3489    120.8690     54.0911   -104.5487
  1600     24.9391     39.0177    103.1407    126.0064     55.3380   -109.6861
  1650     24.9394     40.2647    103.9081    131.1826     56.5850   -114.8623
  1700     24.9396     41.5116    104.6526    136.3967     57.8319   -120.0764
  1750     24.9399     42.7586    105.3755    141.6474     59.0789   -125.3271
  1800     24.9401     44.0056    106.0781    146.9339     60.3259   -130.6135
  1850     24.9403     45.2526    106.7615    152.2548     61.5729   -135.9345
  1900     24.9404     46.4996    107.4266    157.6096     62.8199   -141.2893
  1950     24.9406     47.7466    108.0744    162.9972     64.0669   -146.6769
  2000     24.9407     48.9937    108.7058    168.4167     65.3140   -152.0964
  2100     24.9410     51.4877    109.9228    179.3486     67.8080   -163.0283
  2200     24.9412     53.9818    111.0830    190.3992     70.3021   -174.0789
  2300     24.9414     56.4759    112.1917    201.5634     72.7963   -185.2430
  2400     24.9416     58.9701    113.2532    212.8358     75.2904   -196.5155
  2500     24.9418     61.4642    114.2713    224.2123     77.7845   -207.8920
  2600     24.9419     63.9584    115.2496    235.6887     80.2787   -219.3683
  2700     24.9420     66.4526    116.1909    247.2609     82.7729   -230.9406
  2800     24.9421     68.9468    117.0980    258.9256     85.2671   -242.6053
  2900     24.9422     71.4410    117.9733    270.6794     87.7613   -254.3590
  3000     24.9423     73.9352    118.8188    282.5191     90.2555   -266.1987

Job completed on Wed 18 January 2017 at 12:09:14 æ±äº¬ (æ¨™æº–æ™‚) after 565 s (0:09:25)
